### PR TITLE
Add order items to checkout summary

### PR DESCRIPTION
**Pull request by aleph1ow**
ref: #88 
## Description

Reuse order items view to list items into the checkout summary

## Screenshots (if appropriate):

| Desktop | Mobile |
|--|--|
|screencapture-localhost-3000-checkout-address-2020-11-13-18_50_45|screencapture-localhost-3000-checkout-address-2020-11-13-18_51_26|

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by kennyadsl — APPROVED** [review]
Looks good to me, thanks!

File: app/views/spree/shared/_item_info.html.erb [hunk 1]
```diff
@@ -1,24 +1,30 @@
 <%
```
Comment (ChristianRimondi) [inline app/views/spree/shared/_item_info.html.erb:1]: I'm not sure about moving this partial into the `shared` folder. This folder should be removed in the future, right? (ref #30)

Comment (kennyadsl) [reply app/views/spree/shared/_item_info.html.erb:1]: Good catch!

Comment (aleph1ow) [reply app/views/spree/shared/_item_info.html.erb:1]: Yes, you're right. It's difficult to name a component that you know will be reused in multiple contexts. Should I place it under `components/orders` and reuse it from there?

**Comment by kennyadsl** [comment]
@aleph1ow `fixup` commit  ☝️

**Comment by kennyadsl** [comment]
@aleph1ow Sorry, I've lost track of this one, can you help me figure out what's the status here and if all comments have been addressed?

**Pull request merged**